backend/src/game/game.service.ts

```diff
@@ -405,17 +447,19 @@ export class GameService {
 			return;
 		Logger.log(`${user.username} declined the invitation of ${settings.player1.username}`)
 
-		Logger.log(`decline: gameid=${settings.game.id}`)
 		client.to("game" + settings.game.id.toString()).emit("invitationDeclined", settings)
-		await this.prismaService.userGame.deleteMany({
-			where: {
-				gameId: settings.game.id
-			}
-		})
-		await this.prismaService.game.delete({
-			where: {
-				id: settings.game.id
+		try {
+			await this.prismaService.userGame.deleteMany({
+				where: {
+					gameId: settings.game.id
+				}
+			})
+			await this.prismaService.game.delete({
+				where: {
+					id: settings.game.id
+				}
+			})
 			}
-		})
+		catch {}
```

Comment:
Do we want to do something here ? Even a log ?

Reply:
If the invitation is canceled first, it is not an error that the delete didnt work. 
But we could print something, like `game was already deleted`